src/julia/options.py

```diff
@@ -165,7 +165,7 @@ class JuliaOptions(object):
 
     sysimage = String("sysimage")
     bindir = String("bindir")
-    compiled_modules = Choices("compiled_modules", yes_no_etc())
+    compiled_modules = Choices("compiled_modules", yes_no_etc(), "auto")
```

Comment:
```suggestion
    compiled_modules = Choices("compiled_modules", yes_no_etc("auto"))
```

Reply:
https://github.com/JuliaPy/pyjulia/blob/bfca4a2043229a0bee8d4ea41bfd31e10904e532/src/julia/options.py#L101-L105

I thought that just adds an entry into the dict `"auto": "auto"`